spec/index.bs

```diff
@@ -7555,7 +7555,8 @@ dictionary GPUFragmentState : GPUProgrammableStage {
 
     - |descriptor|.{{GPUFragmentState/targets}}.length must be &le;
         |device|.{{device/[[limits]]}}.{{supported limits/maxColorAttachments}}.
-    - For each non-`null` |colorState| layout descriptor in the list |descriptor|.{{GPUFragmentState/targets}}:
+    - For each index |index| and non-`null` value |colorState| layout descriptor in the list
```

Comment:
nit: This could read like a cartesian product. "For each index index containing a non-null value colorState"?